src/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,33 @@
+import {
+  FormControl,
+  FormLabel,
+  Checkbox as MuiCheckbox,
+  CheckboxProps as MuiCheckboxProps,
+} from "@mui/material";
+
+type CheckboxBaseProps = Pick<
+  MuiCheckboxProps,
+  "checked" | "disabled" | "color" | "id" | "onChange" | "sx" | "value"
+>;
+
+export interface CheckboxProps extends CheckboxBaseProps {
+  label?: string;
+}
+
+const Checkbox = (props: CheckboxProps) => (
+  <FormControl
+    style={{
```

Comment:
So is the idea that we will import all components from agronod now? Previously we have imported most of the base components directly from MUI, for example the Button, Typography and so on and then the baseThem has taken care of the styling with the styleoverrides property, but if we add styling now directly to the components here, it would mean that we will need to change the imports from agronod instead? 

Also it maybe wouldn't make sense to export components with styles and also keep the "stylesovveride" properties in the baseTheme. I guess you have already though about these things but I though its good to ask :)

Reply:
I think this is good point, only really different components from MUI should be imported from storybook, otherwise we should use stylesowerride, I will make sure to fix this :)